Products/ZenModel/ZentinelPortal.py

```diff
@@ -13,49 +13,46 @@
 $Id: ZentinelPortal.py,v 1.17 2004/04/08 15:35:25 edahl Exp $
 """
 
-import urllib, urlparse
+import urlparse
 import re
 import time
 import textwrap
 
 import Globals
-from zExceptions import Redirect
 from AccessControl import getSecurityManager, ClassSecurityInfo
 
 from Products.Sessions.BrowserIdManager import constructBrowserIdManager
-from Products.BeakerSessionDataManager.sessiondata import addBeakerSessionDataManager
+from Products.BeakerSessionDataManager.sessiondata import (
+    addBeakerSessionDataManager
+)
 
 from Products.CMFCore.PortalObject import PortalObjectBase
 from Products.CMFCore.utils import getToolByName
 
 from Products.ZenUtils import Security, Time
-from Products.ZenUtils.Utils import prepId
 from Products.ZenUtils.deprecated import deprecated
-from Products.ZenUI3.browser.interfaces import IErrorMessage
 
+import ZenossSecurity as zs
```

Comment:
The convention in other places has been to import the names.

On a side note, I generally discourage the aliasing of module names unless it's necessary to resolve a name conflict issue.